### PR TITLE
[doc][changelog] 1.4.0-M1 initial changelog

### PR DESCRIPTION
**Pull request by vvlevchenko**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# v1.4.0-M1 (Feb 2020)
+  * 1.4 Language features
+  * klib versioning
+  * commonizer support
+  * caches support
+  * inerrop metadata
```
Comment (sbogolepov) [inline CHANGELOG.md]: Not in M1.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# v1.4.0-M1 (Feb 2020)
+  * 1.4 Language features
+  * klib versioning
+  * commonizer support
+  * caches support
```
Comment (sbogolepov) [inline CHANGELOG.md]: It was added in 1.3.70.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# v1.4.0-M1 (Feb 2020)
+  * 1.4 Language features
+  * klib versioning
+  * commonizer support
```
Comment (SvyatoslavScherbina) [inline CHANGELOG.md]: Including this into K/N changelog is disputable.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+# v1.4.0-M1 (Feb 2020)
```
Comment (SvyatoslavScherbina) [inline CHANGELOG.md]: This one should be clearly marked as milestone (unstable/experimental/development/etc.) version.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,9 @@
...
+  * 1.4 Language features
+  * Klib versioning
+  * Commonizer support
+  * Experimental mimalloc allocator support (-Xallocator=mimalloc) to improve execution time performance
+
```
Comment (SvyatoslavScherbina) [inline CHANGELOG.md]: * Debug improvements?
* https://github.com/JetBrains/kotlin-native/pull/3778
* https://github.com/JetBrains/kotlin-native/pull/3822 and https://github.com/JetBrains/kotlin-native/pull/3842
* https://github.com/JetBrains/kotlin-native/pull/3742

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
+# v1.4.0-M1 (Mar 2020) milestone
+  * Experimental mimalloc allocator support (-Xallocator=mimalloc) to improve execution time performance.
+  * Improveed support for -Xobjc-generics and enabled by default (GH-3778).
```
Comment (SvyatoslavScherbina) [inline CHANGELOG.md:3]: > Improveed

typo

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,8 @@
...
+  * Reworked exception handlings in ObjC/Swing interop.
```
Comment (SvyatoslavScherbina) [inline CHANGELOG.md:4]: > handlings

Probably "handling"

> Swing

:)

Comment (vvlevchenko) [reply CHANGELOG.md]: :)

**Pull request merged**